lib/manager/gradle/index.js

```diff
@@ -5,8 +5,8 @@ const path = require('path');
 const gradle = require('./build-gradle');
 
 const GRADLE_DEPENDENCY_REPORT_COMMAND =
-  'gradle --init-script init.gradle dependencyUpdates -Drevision=release';
-const GRADLE_DEPENDENCY_REPORT_FILENAME = 'build/dependencyUpdates/report.json';
+  'gradle --init-script init.gradle renovate';
+const GRADLE_DEPENDENCY_REPORT_FILENAME = 'renovate.json';
```

Comment:
I think let's make sure it's named something very unique so it doesn't conflict with any regular Renovate config files in the same directory